packages/venia-ui/lib/components/Logo/logo.js

```diff
@@ -1,18 +1,20 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import { mergeClasses } from '../../classify';
+import Image from '../Image';
 import logo from './logo.svg';
 
 const Logo = props => {
     const { height } = props;
     const classes = mergeClasses({}, props.classes);
 
     return (
-        <img
-            className={classes.logo}
-            src={logo}
-            height={height}
+        <Image
             alt="Venia"
+            classes={{ root: classes.logo }}
+            height={height}
+            loading="eager"
```

Comment:
This was more for proof of concept than anything - we don't need `eager` here.

Reply:
Resolved by https://github.com/magento/pwa-studio/pull/1871/commits/a12d03275ea133bea7521ffd077cefb6ca68cbca.